easyconfigs/q/QIIME2/QIIME2-2024.2.eb

```diff
@@ -0,0 +1,31 @@
+# EasyBuild recipy for QIIME2
+# Author: Caspar van Leeuwen
+# SURFsara - The Netherlands
+
+easyblock = 'Conda'
+
+name = 'QIIME2'
+version = '2024.2'
+
+homepage = 'https://qiime2.org/'
+description = """QIIME is an open-source bioinformatics pipeline for performing microbiome analysis
+ from raw DNA sequencing data."""
+
+source_urls = ['https://raw.githubusercontent.com/qiime2/distributions/dev/%(version)s/amplicon/released/']
+sources = ['qiime2-amplicon-ubuntu-latest-conda.yml']
```

Comment:
Please use the versioned file: https://data.qiime2.org/distro/amplicon/qiime2-amplicon-2024.2-py38-linux-conda.yml